activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -93,6 +101,10 @@ def options_include_default?(options)
             options.include?(:default) && !(options[:null] == false && options[:default].nil?)
           end
 
+          def index_in_create(table_name, column_name, options)
```

Comment:
I'm not sure `index_in_create` is an interface of abstract adapter.
If we define `index_in_create` as interface, how about add these comments to this method?

> If the adapter's `supports_indexes_in_create` returns +true+, the adapter should implement + index_in_create+ method.